admin_manual/configuration_server/background_jobs_configuration.rst

```diff
@@ -127,11 +127,14 @@ This approach requires two files: **nextcloudcron.service** and **nextcloudcron.
 
   [Service]
   User=www-data
+  ExecCondition=php -f /var/www/nextcloud/occ status -e
```

Comment:
Good idea :+1: 

https://github.com/nextcloud/server/blob/ad00a149ea2bb262e62fbecf11051fd035f90ec4/cron.php#L45-L52

But isn't the check in cron.php not enough?

Reply:
Not completely. For cron, this will still trigger an email sent to the admin address. But there are also many other reasons you'd want to skip it like in line #130.  Upgrading my instance to use apcu triggered this mail:
```
OCP\HintException: [0]: Memcache OC\Memcache\APCu not available for local cache (Is the matching PHP module installed and enabled?)
```
A temporarily unavailable DB will fail with a stack trace a lot earlier. Manually up/downgrading the instance might temporarily make some files unavailable and trigger a stacktrace. Usually the issue is clear, so the extra noise in the cron/logging should be avoided IMHO.

I've also noticed that I didn't add the check to cron, should I add it?